protocol/contracts/interfaces/IMockFBeanstalk.sol

```diff
@@ -5,6 +5,18 @@ pragma solidity ^0.8.20;
 pragma abicoder v2;
 
 interface IMockFBeanstalk {
+    // Is this the proper location for enums?
```

Comment:
works for me, can we remove comment?

Reply:
Removed here: https://github.com/BeanstalkFarms/Beanstalk/pull/870/commits/649b7baa8a72d31a81b84f6cd421b7e6ae7c3510